pkgs/servers/http/nginx/modules.nix

```
@@ -138,13 +171,14 @@ in
   };
 
   ipscrub = {
-    src = fetchFromGitHub {
-      name = "ipscrub";
-      owner = "masonicboom";
-      repo = "ipscrub";
-      rev = "v1.0.1";
-      sha256 = "0qcx15c8wbsmyz2hkmyy5yd7qn1n84kx9amaxnfxkpqi05vzm1zz";
-    } + "/ipscrub";
+    src = fetchFromGitHub
+      {
```

```suggestion
    src = fetchFromGitHub {
```
Why doesn't it keep this? Strange.

Can you throw the formatting into an extra commit?

---

Sure thing.